benchmarks: check deployment files prior to tests/use.

klihub opened the pull request:
Check the existence of deployment files before running any tests. If a deployment file is simply set to "1" or "yes", try picking up the corresponding generated file from its stock location, assuming we're running out of a cloned repository.

jukkar reviewed (commented):
Looks good, minor nit about commit msg. The previous commits related to this feature are now named "test/statistics-analysis: ...." perhaps we could follow suit here so the commits are easier to find later (if needed).

fmuyassarov commented on test/statistics-analysis/scripts/run-tests.sh:
```diff
@@ -47,6 +47,34 @@ get_pod_name() {
+    if [ "$name" == "baseline" ]; then
+	return 0
+    fi
+    if [ -z "$deployment" ]; then
+	return 0
+    fi
+
+    if [ -f "$deployment" ]; then
+	return 0
+    fi
```
nit, can we perhaps merge these two operators?
```suggestion
    if  [ -z "$deployment" ] || [ -f "$deployment" ]; then
	return 0
    fi
```

klihub replied:
Updated.

klihub commented:
> Looks good, minor nit about commit msg. The previous commits related to this feature are now named "test/statistics-analysis: ...." perhaps we could follow suit here so the commits are easier to find later (if needed).

Fair enough. I updated the commit message. Also added another commit which verifies that known dependencies are present  (python3, nc, envsubst) and tests that the plotter can import its python3 dependencies.

fmuyassarov reviewed (approved):
LGTM
